.github/workflows/main.yml

```diff
@@ -51,3 +51,19 @@ jobs:
       run: |
         bazel build //typescript/...
         bazel test //typescript/...
+
+    - name: Building //csharp directory (Linux)
+      if: runner.os == 'Linux'
+      run: |
+        bazel build \
+        --host_platform @io_bazel_rules_dotnet//dotnet/toolchain:linux_amd64_3.1.407 \
+        --platforms @io_bazel_rules_dotnet//dotnet/toolchain:linux_amd64_3.1.407 \
```

Comment:
What happens when we build without these flags? Ideally they should not be needed, and at least one of the toolchains registered matches the default platform.

Reply:
According to the documentation, https://tomaszstrejczek.github.io/rules_dotnet/, it is mandatory to add the flags. Without them it fails in CI and locally.

Reply:
OK, fair enough. I have some objections to the way the rules are written, but we can't do anything about that here. It looks like their toolchains require a version constraint to be set on both execution and target platforms. The default platforms have no version constraint, so one must be defined and specified.